app/scripts/modules/core/pipeline/config/stages/createLoadBalancer/createLoadBalancerExecutionDetails.html

```diff
@@ -0,0 +1,60 @@
+<div ng-controller="createLoadBalancerExecutionDetailsCtrl">
+    <execution-details-section-nav sections="configSections"></execution-details-section-nav>
+    <div class="step-section-details" ng-if="detailsSection === 'loadBalancerConfig'">
+        <div class="row">
+
```

Comment:
If you include a `<div class="col-md-12">` tag here (and close it after the `</table>`), you should have some padding between the details section and the table contents, which (I am guessing) would make it align better with the tab heading (Load Balancer Config).